tests/functional/test_colorama.py

```diff
@@ -0,0 +1,222 @@
+import os
+import sys
+from contextlib import contextmanager
+
+from mock import Mock, patch
+
+from awscli.testutils import unittest, skip_if_windows
+from awscli.testutils import capture_output
+from awscli.compat import StringIO
+
+import colorama
+from colorama import Fore
+from colorama import Back
+from colorama.winterm import WinTerm, WinStyle, WinColor
+
+
+@skip_if_windows('Posix color code tests')
+class TestPosix(unittest.TestCase):
+    _COLORAMA_KWARGS = {
```

Comment:
What do you think about having a constant in table.py for these colorama kwargs?  Seems like this can get out of sync if we change the kwargs we use in the table formatter and we may no longer be testing how we're using colorama.